src/main.ts

```diff
@@ -327,6 +328,78 @@ export async function getExecutionGraphResult(
   }
 }
 
+export function prettifyExecutionGraphResult(
+  executionGraphResult: Object
+): void {
+  core.info(
+    ansi.bold(
+      `Execution Graph Result: ${
+        executionGraphResult["passed"]
+          ? ansi.green("passed")
+          : ansi.red("failed")
+      }`
+    )
+  )
+  let actionsPassed = 0
+  let actionsFailed = 0
+  let actionsSkipped = 0
+  for (const task of executionGraphResult["actions"]) {
+    if (task["passed"]) {
+      actionsPassed++
+    } else {
```

Comment:
You probably can use something like this to detect the skipped actions. ( not meant to be a straight replacement )
```suggestion
    } else if (task["status"] === "SKIPPED") {
      actionsSkipped++
    } else {
      actionsFailed++
    }
```